content/sensu-go/5.15/installation/auth.md

```diff
@@ -30,7 +30,7 @@ For Sensu's [default user credentials][3] and more information about configuring
 In addition to built-in RBAC, Sensu includes [license-activated][6] support for authentication using external authentication providers.
 Sensu currently supports Microsoft Active Directory and standards-compliant Lightweight Directory Access Protocol tools like OpenLDAP.
 
-**LICENSED TIER**: Unlock authentication providers in Sensu Go with a Sensu license. To activate your license, see the [getting started guide][6].
+**LICENSED**: Unlock authentication providers for more than 100 entities with a Sensu license. To activate your license, see the [getting started guide][6].
```

Comment:
This is an example of how I'd like to rephrase the "LICENSED TIER" notes throughout the docs. WDYT? I can propagate the change throughout the 5.15 docs when final.